.github/workflow/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 12.x
+          - 14.x
+          - 15.x
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm test
+      - run: npm run doc
```

Comment:
This command does not work locally for me as well. We can leave it out for now and fix it seperately.

Reply:
ok, if we leave it out we will loose content on http://linkeddata.github.io/rdflib.js/doc/. I will try to see what is wrong with the command

Reply:
fixed in https://github.com/linkeddata/rdflib.js/pull/522